docs/en/patterns/muted-heading.md

```diff
@@ -5,15 +5,15 @@ title: Muted heading
 
 ## Muted heading
 
-A Muted heading can be used to introduce a collection of icons or images.
+<hr>
+
+A muted heading can be used to introduce a collection of icons or images.
```

Comment:
We use it elsewhere. Perhaps we should rephrase this as something like:

"A muted heading can be used to subtly provide context to content without the heading itself being too prominent"

I'm not sure that that's the best wording.

Reply:
I'm ok with that, reads better than what I put together 😆 will update.